front/app/containers/Admin/sideBar/MenuItem.tsx

```diff
@@ -124,46 +116,26 @@ const MenuItem = ({ navItem }: Props) => {
   }
 
   return (
-    <Tooltip
-      content={
+    <MenuItemLink
+      to={navItem.link}
+      className={`intercom-admin-menu-item-${navItem.name}`}
+    >
+      <>
         <Box
           display="flex"
-          flexDirection="column"
+          flex="0 0 auto"
           alignItems="center"
-          gap="20px"
-          p="8px"
+          justifyContent="center"
+          className={navItem.iconName}
         >
-          <Image src={tooltipImage} alt="" w="250px" />
-          <FormattedMessage {...messages.proposalsTooltip} />
```

Comment:
This message (app.containers.AdminPage.SideBar.proposalsTooltip) can also be deleted. :)